app/javascript/stylesheets/application.scss

```diff
@@ -1,6 +1,25 @@
 @import "~bootstrap/scss/bootstrap";
 @import "./_sidebar";
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
 
 #page-content-wrapper {
   width: 100%;
 }
+
+.big-button {
+  width: 100%;
+}
+
+.column-background {
+  display: flex;
```

Comment:
Since this is styling for a specific element, consider using an ID selector, rather than class selector. E.g. '#introduction-image' or whatever represents its purpose better.

Reply:
actually, I use this element twice, in different views - is it still better to use an ID in such case?